### DISCUSSION TITLE
0.4.0

### DISCUSSION THREAD

**Discussion by mgmeyers** [mgmeyers/obsidian-kanban#124] [Releases]
- Adds support for pulling in metadata from linked pages
- Adds support for pulling tags out of card titles and displaying them below dates

<img width="975" alt="Screen Shot 2021-06-01 at 7 23 51 PM" src="https://user-images.githubusercontent.com/2694747/120414500-ef7f7c00-c30e-11eb-8f97-2b125cc467dc.png">

<img width="290" alt="Screen Shot 2021-06-01 at 7 19 39 PM" src="https://user-images.githubusercontent.com/2694747/120414117-52244800-c30e-11eb-8022-217643cebe32.png">

<hr /><em>This discussion was created from the release <a href='https://github.com/mgmeyers/obsidian-kanban/releases/tag/0.4.0'>0.4.0</a>.</em>

**Comment by danieltomasz** [comment]
Wow! Great release!

**Comment by onedeneb** [comment]
Great job! Thank for you making this.

**Comment by KyleSQ9** [comment]
I'm a novice to Obsidian, can somebody explain to me what metadata keys are? I'm trying to have my cards display the description of the notes but it doesn't seem to work. Thanks!

**Reply by danieltomasz** [reply]
Hi! Metada are part that could be put into YAML part of the file (you need to define it by yourself when creating note)
https://help.obsidian.md/Advanced+topics/YAML+front+matter. 
Then you could choose the preferred keys in Kanban settings.
If you insert your note into card using [[ wikilinks]] it should show the keys specified by you.

**Reply by danieltomasz** [reply]
@qnine if you need any help in other issues  try a Obsidian  discord, you will  usually get answer to your questions really fast from other users

**Comment by fuhadabdulla** [comment]
Thank you so much for developing this plugin. It is just awesome . Just loving it

**Reply by PENorris** [reply]
This is very cool.   Thoughts on being able to eventually edit these items from the board itself?
